### DISCUSSION TITLE
New version (at least python)

### DISCUSSION THREAD

**Discussion by ggsdc** [google/or-tools#3935] [Q&A] [answered]
Hi!

With the issue reported here: https://github.com/google/or-tools/issues/3889

I can see that it has already been fixed but no new version has been deployed to pypi. Is there any plans to deploy a new version to pypi or should we add pandas as a requirement to our projects to avoid errors on deployment for the time being?

Thanks in advance.

**Comment by Mizux** [comment, marked as answer]
We hope to release a new version mid/end october if everything go well.. Before release we still need:
* finish constraint solver python unit tests merge with google3 (WIP)
* finish math_opt integration in CMake (C++ only first) (WIP)
* bump SCIP to v804 (todo)
* bump PDLP (todo)
